staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -84,6 +84,15 @@ type KubeSchedulerConfiguration struct {
 	// If this value is nil, the default value will be used.
 	BindTimeoutSeconds *int64 `json:"bindTimeoutSeconds"`
 
+	// PodInitialBackoffDurationSeconds is the initial backoff duration for unschedulable pods.
+	// It should be greater than 0. If this value is nil, the default value will be used.
```

Comment:
document the default

Reply:
"If specified, it must be greater than 0"

Reply:
I am assuming changing the default value (if it ever happens) is considered an API change and so documenting it here is necessary as it is part of the API contract?

Reply:
correct. changing the default should not be done in the same version... it changes behavior release-to-release for someone using an identical config